Emby.Server.Implementations/HttpServer/FileWriter.cs

```diff
@@ -145,8 +148,7 @@ private void SetRangeValues()
             }
         }
 
-        private string[] SkipLogExtensions = new string[]
-        {
+        private readonly string[] SkipLogExtensions = {
```

Comment:
make it static and camelBack with `_` prefixed

Reply:
Static readonly should be PascalCase according to Rider.